pylti/flask.py

```diff
@@ -192,7 +195,7 @@ def _check_role(self):
             role = self.lti_kwargs['role']
         log.debug("check_role lti_role={} decorator_role={}"
                   .format(self.role, role))
-        if not self.is_role(role):
+        if not LTI.is_role(role):
```

Comment:
You should still use `self.is_role` here since it isn't a class method, but a static method, and if we ever needed to add self, we wouldn't have to change these calls.